### PR TITLE
Release/1.16.0

### PR DESCRIPTION
**Pull request by bingzhang**
## Description
_Please provide a summary of the pull request and the issue it fixes. Please add necessary details, context, dependencies, explanation of when review is needed (see next section), etc._

**Fixes #(add issue number here and remove parentheses)**

[comment]: # (This project only accepts pull requests related to open issues.)
[comment]: # (If suggesting a new feature or change, please discuss it in an issue first.)
[comment]: # (If fixing a bug, there should be an issue describing it with steps to reproduce.)

## Review Time Estimate
_Please give your idea of how soon this pull request needs to be reviewed by selecting one of the options below. This can be based on the criticality of the issue at hand and/or other relevant factors._

[comment]: # (To select an option, please put an 'x' in the applicable box.)
[comment]: # (If you're unsure about any of these, don't hesitate to ask. We're here to help!.)

- [ ] Immediately
- [ ] Within a week
- [ ] When possible

## Type of changes
_Please select a relevant option:_

[comment]: # (To select an option, please put an 'x' in the applicable box.)
[comment]: # (If you're unsure about any of these, don't hesitate to ask. We're here to help!.)

- [ ] Bug fix (non-breaking change which fixes an issue).
- [ ] New feature (non-breaking change which adds functionality).
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected).
- [ ] Other (any another change that does not fall in one of the above categories.)

## Checklist:
_Please select all applicable options:_

[comment]: # (To select your options, please put an 'x' in the all boxes that apply.)
[comment]: # (If you're unsure about any of these, don't hesitate to ask. We're here to help!.)

- [ ] I have signed the Rokwire Contributor License Agreement ([CLA](https://rokwire.org/rokwire_cla)). (_Any contributor who is not an employee of the University of Illinois whose official duties include contributing to the Rokwire software, or who is not paid by the Rokwire project, needs to sign the CLA before their contribution can be accepted._)
- [ ] I have updated the [CHANGELOG](../CHANGELOG.md).
- [ ] I have read the [Contributor Guidelines](../CONTRIBUTING.md).
- [ ] I have performed a self-review of my own code.
- [ ] I have commented my code, particularly in hard-to-understand areas.
- [ ] My change requires updating the documentation.
- [ ] I have made necessary changes to the documentation.
- [ ] I have added tests related to my changes.
- [ ] My changes generate no new warnings.
- [ ] New and existing unit tests pass locally with my changes.
- [ ] Any dependent changes have been merged and published in downstream modules.

---

[comment]: # (Template credit: This pull request template is based on Embedded Artistry {https://github.com/embeddedartistry/templates/blob/master/.github/PULL_REQUEST_TEMPLATE.md}, Clowder {https://github.com/clowder-framework/clowder/blob/develop/.github/PULL_REQUEST_TEMPLATE.md}, and TalAter {https://github.com/TalAter/open-source-templates} templates.)

### CODE REVIEW

File: SECURITY.md [hunk 1]
```diff
@@ -4,8 +4,9 @@
 
 Patches for **Rokwire Building Blocks** in this repository will only be applied to the following versions:
 
-| Version | Supported          |
-| ------- | ------------------ |
+| Version  | Supported          |
+|----------| ------------------ |
+| 1.15.0   | :white_check_mark: |
```
Comment (sandeep-ps) [inline SECURITY.md:9]:
```suggestion
| 1.16.0   | :white_check_mark: |
| 1.15.0   | :white_check_mark: |
```

File: contributions/contribution.yaml [hunk 2]
```diff
@@ -2,7 +2,7 @@ openapi: 3.0.0
 info:
   title: Rokwire Contributions Building Block API
   description: Contributions Building Block API Documentation
-  version: 1.14.1
+  version: 1.15.0
```
Comment (sandeep-ps) [inline contributions/contribution.yaml:5]: Please update the version in the YAML file as the next deployment of these modules will need the latest version. Thanks.

File: eventservice/events.yaml [hunk 3]
```diff
@@ -2,7 +2,7 @@ openapi: 3.0.0
 info:
   title: Rokwire Events Building Block API
   description: Events Building Block API Documentation
-  version: 1.14.1
+  version: 1.15.0
```
Comment (sandeep-ps) [inline eventservice/events.yaml:5]: Please update the version in the YAML file as the next deployment of these modules will need the latest version. Thanks.

File: loggingservice/logging.yaml [hunk 4]
```diff
@@ -2,7 +2,7 @@ openapi: 3.0.0
 info:
   title: Rokwire Logging Building Block API
   description: Logging Building Block API Documentation
-  version: 1.14.1
+  version: 1.15.0
```
Comment (sandeep-ps) [inline loggingservice/logging.yaml:5]: Please update the version in the YAML file as the next deployment of this module will need the latest version. Thanks.

File: CHANGELOG.md [hunk 5]
```diff
@@ -501,7 +518,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Removed
 - References to AWS keys and variables in the Events Building Block.
 
-[Unreleased]: https://github.com/rokwire/rokwire-building-blocks-api/compare/1.15.1...HEAD
+[Unreleased]: https://github.com/rokwire/rokwire-building-blocks-api/compare/1.16.1...HEAD
```
Comment (sandeep-ps) [inline CHANGELOG.md:521]:
```suggestion
[Unreleased]: https://github.com/rokwire/rokwire-building-blocks-api/compare/1.16.0...HEAD
```

File: CHANGELOG.md [hunk 6]
```diff
@@ -501,7 +518,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
...
+[1.16.0]: https://github.com/rokwire/rokwire-building-blocks-api/compare/1.15.1...1.16.0
```
Comment (sandeep-ps) [inline CHANGELOG.md:522]:
```suggestion
[1.16.0]: https://github.com/rokwire/rokwire-building-blocks-api/compare/1.15.0...1.16.0
```

File: SECURITY.md [hunk 7]
```diff
@@ -6,7 +6,12 @@ Patches for **Rokwire Building Blocks** in this repository will only be applied
 
 | Version  | Supported          |
 |----------| ------------------ |
+<<<<<<< HEAD
+| 1.16.0   | ✅ |
```
Comment (sandeep-ps) [inline SECURITY.md:10]: Please update the security.MD to use the following for the first two versions:
```
| 1.16.0   | :white_check_mark: |
| 1.15.0   | :white_check_mark: |
```
It looks like currently, it's not using markdown code. Maybe my suggestion automatically converted it to an icon? Sorry if that's the case.

Also, please remove the markers from the merge conflict. Thanks.

File: CHANGELOG.md [hunk 8]
```diff
@@ -501,7 +518,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Removed
 - References to AWS keys and variables in the Events Building Block.
 
-[Unreleased]: https://github.com/rokwire/rokwire-building-blocks-api/compare/1.15.1...HEAD
+[Unreleased]: https://github.com/rokwire/rokwire-building-blocks-api/compare/1.16.0...HEAD
+[1.16.0]: https://github.com/rokwire/rokwire-building-blocks-api/compare/1.15.1...1.16.0
```
Comment (sandeep-ps) [inline CHANGELOG.md:522]:
```suggestion
[1.16.0]: https://github.com/rokwire/rokwire-building-blocks-api/compare/1.15.0...1.16.0
```
We don't have version 1.15.1, right?

**Comment by sandeep-ps** [comment]
I tested this branch with Postman tests collection, and the tests look fine. Please address the remaining comments in this PR, and I hope we can merge this soon. Thanks.

**Comment by sandeep-ps** [comment]
Hi, @bingzhang, Could you please go through the remaining open comments in this PR and let me know when it's ready for review again? Thanks.

**Review by sandeep-ps — APPROVED** [review]
Approving. Please update the version number in appconfig.yaml too. Thanks.

**Comment by sandeep-ps** [comment]
I have updated the tasks linked to this PR.

**Comment by sandeep-ps** [comment]
I thought I would update the YAML file and merge this. Thanks.

**Comment by sandeep-ps** [comment]
@minump, any quick comments? We will not be requesting an immediate deployment of the Catalog or the Contributions Building Block, unless you feel the need. We can do another release after the other open PRs are reviewed and merged. I am revisiting those. Thanks.

**Review by sandeep-ps — APPROVED** [review]

**Review by minump — APPROVED** [review]
Approved.

File: CODEOWNERS [hunk 9]
```diff
@@ -15,14 +15,14 @@
...
 
 # Profile Building Block
-/profileservice/ @ywkim312
+/profileservice/ @sandeep-ps
 
 # Contributions Building Block
-/contributions/* @ywkim312
-/contributions/api/ @ywkim312
+/contributions/* @minump
+/contributions/api/ @minump
```
[the remark this answers is not in the document]
Comment (sandeep-ps) [reply CODEOWNERS:25]: Let's keep it as these would have to move to different repos.

**Pull request merged**